scripts/systemd/etc/sysconfig/fofb-epics-ioc-slot-mapping

```diff
@@ -2077,300 +2077,249 @@ CRATE_21_FOFB_23_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
 CRATE_21_FOFB_24_PV_AREA_PREFIX=XX-21SL24
 CRATE_21_FOFB_24_PV_DEVICE_PREFIX=RaBPM:BS-FOFBCtrl:
 
+# --- CRATE 22 (Homolog) ---
 
-# --- CRATE 22 ---
+# Crate 22 - FOFB slot 1 - DE-22SL01:RaBPM:BS-FOFBCtrl
```

Comment:
I was thinking `DE-22SL01:BS-FOFBCtrl`. Do you think we need the `RaBPM`part?

Reply:
No, I don't, just kept it bc all the other macros have...

Reply:
I suggest to append `RaBPM` to all `*FOFB_{3,5}_PV_AREA_PREFIX*` and remove it from all `*PV_DEVICE_PREFIX`. By doing that we don't break any PV name being used and also keep consistency with [tim-rx-epics-ioc-slot-mapping](https://github.com/lnls-dig/tim-rx-epics-ioc/blob/fed3ea3bdb13bc083d79e0fc1884dc55546a6c7c/scripts/systemd/etc/sysconfig/tim-rx-epics-ioc-slot-mapping). What do you think?

Reply:
Addressed on 2dd6dc49f41c519e628355aec210a5f7c2483181.